From patchwork Thu Dec 17 02:56:09 2020
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
X-Patchwork-Submitter: Stephen Rothwell <sfr@canb.auug.org.au>
X-Patchwork-Id: 11979025
Return-Path: <kvm-owner@kernel.org>
X-Spam-Checker-Version: SpamAssassin 3.4.0 (2014-02-07) on
	aws-us-west-2-korg-lkml-1.web.codeaurora.org
X-Spam-Level: 
X-Spam-Status: No, score=-10.8 required=3.0 tests=BAYES_00,DKIM_SIGNED,
	DKIM_VALID,DKIM_VALID_AU,HEADER_FROM_DIFFERENT_DOMAINS,INCLUDES_PATCH,
	MAILING_LIST_MULTI,SPF_HELO_NONE,SPF_PASS autolearn=unavailable
	autolearn_force=no version=3.4.0
Received: from mail.kernel.org (mail.kernel.org [198.145.29.99])
	by smtp.lore.kernel.org (Postfix) with ESMTP id 5C3A0C1B0D8
	for <kvm@archiver.kernel.org>; Thu, 17 Dec 2020 02:57:12 +0000 (UTC)
Received: from vger.kernel.org (vger.kernel.org [23.128.96.18])
	by mail.kernel.org (Postfix) with ESMTP id 2084023406
	for <kvm@archiver.kernel.org>; Thu, 17 Dec 2020 02:57:12 +0000 (UTC)
Received: (majordomo@vger.kernel.org) by vger.kernel.org via listexpand
        id S1727620AbgLQC44 (ORCPT <rfc822;kvm@archiver.kernel.org>);
        Wed, 16 Dec 2020 21:56:56 -0500
Received: from bilbo.ozlabs.org ([203.11.71.1]:59597 "EHLO ozlabs.org"
        rhost-flags-OK-OK-OK-OK) by vger.kernel.org with ESMTP
        id S1727115AbgLQC44 (ORCPT <rfc822;kvm@vger.kernel.org>);
        Wed, 16 Dec 2020 21:56:56 -0500
Received: from authenticated.ozlabs.org (localhost [127.0.0.1])
        (using TLSv1.3 with cipher TLS_AES_256_GCM_SHA384 (256/256 bits)
         key-exchange ECDHE (P-256) server-signature RSA-PSS (4096 bits)
 server-digest SHA256)
        (No client certificate requested)
        by mail.ozlabs.org (Postfix) with ESMTPSA id 4CxGpg5JCjz9sRR;
        Thu, 17 Dec 2020 13:56:11 +1100 (AEDT)
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/simple; d=canb.auug.org.au;
        s=201702; t=1608173773;
        bh=0BUN17/xDSq7+CWgU8wi0dZDg7QZRtEzB1WlEheKggE=;
        h=Date:From:To:Cc:Subject:From;
        b=TpWRTGPMExPzxK4g6VJFBkIPYYtyYT4sVp8ZeelQHmrO+qbYC7mvlB2fWDyd++g7k
         NQWfs75u66yPdiG8dTWmlqfSj0vmQt0SYH6dNZZXcUgiRjf03597lrr6HmScTzLG6b
         jdF8cQEPnVlA4XKrndHBSu1ozFroi5JlXplqHyM28V6SS4LsYV/W5s1NBqgiEGjpk/
         knO6reWL99Zo6oC0EKRMc8QeZim3R0VLRrYw9YmvBB4jv2B2N+m3K3glziFJ/P97wz
         P/+M5WCjpv5ZW072G9zJ8x4Yv9s/l88nKMONdppCsMxrgivTukK2WXByGn0RMXBZ3n
         9xmcNUdmuRO8g==
Date: Thu, 17 Dec 2020 13:56:09 +1100
From: Stephen Rothwell <sfr@canb.auug.org.au>
To: Paolo Bonzini <pbonzini@redhat.com>, KVM <kvm@vger.kernel.org>
Cc: Chen Zhou <chenzhou10@huawei.com>,
        Linux Kernel Mailing List <linux-kernel@vger.kernel.org>,
        Linux Next Mailing List <linux-next@vger.kernel.org>,
        Tom Lendacky <thomas.lendacky@amd.com>
Subject: linux-next: manual merge of the kvm tree with Linus' tree
Message-ID: <20201217135609.6543533c@canb.auug.org.au>
MIME-Version: 1.0
Precedence: bulk
List-ID: <kvm.vger.kernel.org>
X-Mailing-List: kvm@vger.kernel.org

Hi all,

Today's linux-next merge of the kvm tree got a conflict in:

  arch/x86/kvm/svm/svm.c

between commit:

  054409ab253d ("KVM: SVM: fix error return code in svm_create_vcpu()")

from Linus' tree and commit:

  add5e2f04541 ("KVM: SVM: Add support for the SEV-ES VMSA")

from the kvm tree.

I fixed it up (see below) and can carry the fix as necessary. This
is now fixed as far as linux-next is concerned, but any non trivial
conflicts should be mentioned to your upstream maintainer when your tree
is submitted for merging.  You may also want to consider cooperating
with the maintainer of the conflicting tree to minimise any particularly
complex conflicts.

diff --cc arch/x86/kvm/svm/svm.c
index da7eb4aaf44f,941e5251e13f..000000000000
--- a/arch/x86/kvm/svm/svm.c
